Ports/lite-xl/package.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=lite-xl
+version=2.1.0
+useconfigure=true
```

Comment:
Please quote all of these with single quotes, or double quotes when using string interpolation.